src/tooltip.tsx

```diff
@@ -112,386 +161,442 @@ const Tooltip: React.FC<Props> = ({
-                            }
-                        }}
-                        onPointerLeave={
-                            isTouchableDevice
-                                ? noOp
-                                : () => {
-                                      closeTooltipTimeoutId.current = setTimeout(() => {
-                                          if (!isPointerOver.current) return;
-
-                                          closeTooltipTimeoutId.current = null;
-                                          isPointerOver.current = false;
-                                          toggleVisibility();
-                                      }, 100);
-                                  }
-                        }
-                    >
-                        <div
-                            style={getArrowStyles(position)}
-                            className={classnames(styles.arrowWrapper, arrowClassNameByPosition[position])}
-                        >
-                            <div className={styles.arrow} />
-                        </div>
-                        {(title || description) && (
-                            <Stack space={4}>
-                                {title && <Text2 medium>{title}</Text2>}
-                                {description && <Text2 regular>{description}</Text2>}
-                            </Stack>
-                        )}
-                        {extra || children}
-                    </div>
-                </CSSTransition>
+                    {(transitionStatus) => {
+                        return (
+                            <div
+                                style={{
+                                    /**
+                                     * Hack to prevent text from wrapping automatically when touching the viewport's edges,
+                                     * even if the content's width didn't reach the max width.
+                                     * https://stackoverflow.com/questions/66106629/how-to-disable-text-wrapping-when-viewport-border-is-reached
+                                     */
+                                    width: `calc(100vw + ${TOOLTIP_MAX_WIDTH}px)`,
+                                    top: 0,
+                                    left: 0,
+                                    position: 'fixed',
+                                    visibility: tooltipComputedProps ? 'visible' : 'hidden',
+                                    ...assignInlineVars({
+                                        ...(tooltipComputedProps
+                                            ? {
+                                                  [styles.tooltipVars.top]: `${tooltipComputedProps.top}px`,
+                                                  [styles.tooltipVars.left]: `${tooltipComputedProps.left}px`,
+                                                  [styles.tooltipVars.padding]: tooltipComputedProps.padding,
```

Comment:
why don't just use inline styles? why do you need the css vars?

Reply:
It was like this in previous iterations. After changing it many times, I didn't realize I could remove the css vars. Updated